source/gui/settingsDialogs.py

```diff
@@ -1125,6 +1125,8 @@ def _updateCurrentMirrorURL(self):
 
 	def postSave(self):
 		if self.oldLanguage != config.conf["general"]["language"]:
+			config.conf["braille"]["translationTable"] = "auto"
```

Comment:
Honestly, I consider this to be a bit aggressive actually. I sometimes change my NVDA language from Dutch to English, and then my braille tables will get changed without notice.